Formula/luaradio.rb

```diff
@@ -0,0 +1,47 @@
+class Luaradio < Formula
+  desc "lightweight, embeddable flow graph signal processing framework for SDR"
+  homepage "http://luaradio.io/"
+  url "https://github.com/vsergeev/luaradio/archive/v0.1.0.tar.gz"
+  sha256 "4423f94c0bb259952eae7db11e4db4509dfa7c6e205798c35a095050c530212b"
+
+  depends_on "pkg-config" => :build
+  depends_on "luajit"
+  depends_on "fftw" => :recommended
+
+  def install
+    cd "embed" do
+      system "make", "install", "PREFIX=#{prefix}"
+    end
+  end
+
+  test do
+    # Create text file which will be read and processed by LuaRadio
```

Comment:
> Where's that written? I did not find that in the contribution guideline or the code of conduct.

It's not written; it goes without saying on pretty much any project on GitHub (or open-source in general) that if changes are requested in a PR and you aren't willing to make them: your PR will not be merged.



> The comments where placed to make that somewhat clear and therefore an extra file is needed.

Sure but for those who maintain Homebrew these comments are self-evident from the code. Comments should describe why something is done, not describe what the code does.



> Honestly I feel a bit upset that I have to justify myself for my opinion on such a trivial issue and I'm again disappointed that I'm getting encountered in such a rough way as to me it seems like your opinion the only one that matters.

Well, two maintainers have asked you to do this and, when it comes to maintaining software in Homebrew I'm afraid our opinions do matter more than yours do. We've been running this project (some would say successfully) for a long time and you're not being treated differently to any other PR.



> I like to enable new tools to be available through Homebrew but this nitpicking on trivial, non-blocking issues bothers me almost every time in a way that I want to quit doing that.

I also like new tools being available. It is a blocking issue to this PR being merged otherwise we wouldn't have asked you to do it. Our review process is thorough because when things are merged we're responsible for maintaining them and when we allow bad code into Homebrew it lowers the overall standard.

I hope that adds some explanation for this. If that's sufficient, please remove the comment. If it's not, please close this PR or indicate you'd like us to close it. Thanks!